pkg/flowaggregator/flowaggregator_test.go

```diff
@@ -66,9 +66,29 @@ func TestFlowAggregator_sendFlowKeyRecord(t *testing.T) {
 		registry:                    mockIPFIXRegistry,
 		set:                         mockDataSet,
 		flowAggregatorAddress:       "",
+		includePodLabels:            true,
 		observationDomainID:         testObservationDomainID,
 		podInformer:                 informerFactory.Core().V1().Pods(),
 	}
+
+	faWithoutPodLabels := &flowAggregator{
```

Comment:
Could we have a for loop here with true or false `includePodLabels` when creating flowAggregator object instead of copying these lines?

Reply:
Sounds good. Done.